BetaCameras/Kinect2/Kinect2.csproj

```diff
@@ -34,12 +34,37 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'ReleaseStrongName|AnyCPU'">
+    <OutputPath>..\..\bin\x64\ReleaseStrongName\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Optimize>true</Optimize>
+    <DebugType>pdbonly</DebugType>
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
> Shouldn't the `PlatformTarget` be "x64"?

I think the error is in the output directory - it should be AnyCPU. It seems like the Kinect2 can run in 32 bit applications: https://stackoverflow.com/questions/37697360/32-bit-support-kinect-2-sdk

> Does it make sense to have the MetriCam2 assembly in AnyCPU when we only have x64 cameras?

Currently, we only deploy 64 bit implementations, but it is not sure that it will stay like that. It wasn't that long ago that we had to deliver a 32 bit library for a PMD camera. But from my point of view we could skip the x86 implementations as long as we do not need one (means having a x86 configuration in the solution, but not for new camera projects).

> The only reason I could think of would be Designer compatibility. Is this still relevant?

I don't know - but what's wrong having MetriCam in AnyCPU (except maybe a more manageable Jenkinsfile)?